docs/docs/gatsby-link.md

```diff
@@ -297,17 +297,13 @@ following may be a good starting point:
 
 ```jsx
 import { Link as GatsbyLink } from "gatsby"
```

Comment:
I'm not completely sure why `Link` is used here.

why not `import GatsbyLink from 'gatsby-link'`?

Reply:
Because `gatsby-link` is internal dependency of gatsby (is not installed by users directly). Importing modules that are not direct dependencies of your packages (site in this case) is bad practice and can lead to very unexpected results (specifically if there would be more than 1 instance of `gatsby-link` installed in `node_modules)

Reply:
Because of that - the `isInternalLink` would need to be re-exported from `gatsby` (similiar to how `Link` is re-exported from `gatsby`)

Reply:
oh, gotcha.

What do you think expose is using `gatsby.Link`?

```
import { Link as GatsbyLink } from "gatsby"
const {isInternalLink} = GatsbyLink
```